future-apps/payroll/app/src/utils/formatting.js

```diff
@@ -1,15 +1,26 @@
 import { format as dateFormatter } from 'date-fns'
+import { round } from './math-utils'
 
 const DEFAULT_DATE_FORMAT = 'LL/dd/yyyy'
 
-const currencyFormatter = new Intl.NumberFormat('en-US', {
-  style: 'currency',
-  currency: 'USD',
-  minimumFractionDigits: 2
-})
-
-export const formatCurrency = currencyFormatter.format
+export const SECONDS_IN_A_YEAR = 31557600 // 365.25 days
 
 export function formatDate (date, format = DEFAULT_DATE_FORMAT) {
   return dateFormatter(date, format)
 }
+
+const formatter = new Intl.NumberFormat('en-US', {
+  minimumFractionDigits: 0
+})
+
+export function formatCurrency (
+  amount,
+  symbol,
+  decimals = 10,
+  pow = 18,
+  rounding = 2
+) {
+  const number = round(((amount / Math.pow(decimals, pow)) * SECONDS_IN_A_YEAR), rounding)
```

Comment:
I think we shouldn't use SECONDS_IN_A_YEAR here, we may need this function for formatting other amounts than annual salary

Reply:
done!